examples/practice-typescript/src/components/Input/index.scss

```diff
@@ -0,0 +1,17 @@
+.search {
+  border: 1px solid $border-search-color;
+  border-radius: 4px;
+  outline: none;
+  max-width: 228px;
+  max-height: 40px;
+  margin-right: 85px;
+  padding: 11px 10px 10px 40px;
+  font-size: 16px;
+}
+
+.search-icon {
+  background-image: url(../assets/images/managers/icon-search.png);
```

Comment:
But why it is a background image? When a thing is a background and is it a background here?